just_audio/lib/just_audio.dart

```diff
@@ -584,6 +584,23 @@ class AudioPlayer {
     await (await _platform).setVolume(SetVolumeRequest(volume: volume));
   }
 
+  /// Sets skipSilence option on exoPlayer to true/false. Works only on Android for now.
+  Future<void> setSkipSilence(bool enabled) async {
+    if (_disposed) return;
+    enabled ??= false;
+    await (await _platform).setSkipSilence(SetSkipSilenceRequest(enabled: enabled));
+  }
+
+  /// Sets volume boost option on exoPlayer to true/false. Works only on Android for now.
+  /// gainMillibels is the value in millibels you want to amplify the volume by.
+  Future<void> setVolumeBoost(bool enabled, {int gainMillibels = 2000}) async {
+    if (_disposed) return;
+    enabled ??= false;
+    if (Platform.isAndroid) {
+      await (await _platform).setVolumeBoost(SetVolumeBoostRequest(enabled: enabled, gainMillibels: gainMillibels));
+    }
```

Comment:
I would rather unconditionally pass this request through to the platform and let it decide what to do with it.